docs/pages/getting-started.html

```diff
@@ -52,6 +52,12 @@ <h3 class="usa-heading">For developers</h3>
 
 <p>The standards are designed to support both older and newer browsers through progressive enhancement, and officially support Internet Explorer 9 and up, and the latest versions of Chrome, Firefox, and Safari. Internet Explorer 8 and below generally see very low usage today, and most agency websites should be able to safely begin support at Internet Explorer 9.</p>
 
+<h4>Progressive enhancement:</h4>
+
+<p>Progressive enhancement is a web design and development strategy emphasizing accessibility through <a href="http://www.w3.org/html/wg/wiki/Guide/Syntax-Semantics" target="_blank" title="World Wide Web Consortium - Guide/Syntax and Semantics">semantic HTML markup</a>. Semantic HTML allows older browsers to present the content of a document to users. The semantic markup is layered with external technologies to enhance the user experience (specifically, CSS and JavaScript), without necessarily effecting the underlying funcitonality of the document and site.</p>
```

Comment:
Josh, I'm getting the same thing. My comment keeps disappearing and
reappearing. Looks like github is having some issues today!

On Mon, May 23, 2016 at 9:02 AM, Josh Bruce <notifications@github.com>
wrote:

> In docs/pages/getting-started.html
> <https://github.com/18F/web-design-standards/pull/1184#discussion_r64244730>
> :
>
> > @@ -52,6 +52,12 @@ <h3 class="usa-heading">For developers</h3>
> >
> >  <p>The standards are designed to support both older and newer browsers through progressive enhancement, and officially support Internet Explorer 9 and up, and the latest versions of Chrome, Firefox, and Safari. Internet Explorer 8 and below generally see very low usage today, and most agency websites should be able to safely begin support at Internet Explorer 9.</p>
> >
> > +<h4>Progressive enhancement:</h4>
> > +
> > +<p>Progressive enhancement is a web design and development strategy emphasizing accessibility through <a href="http://www.w3.org/html/wg/wiki/Guide/Syntax-Semantics" target="_blank" title="World Wide Web Consortium - Guide/Syntax and Semantics">semantic HTML markup</a>. Semantic HTML allows older browsers to present the content of a document to users. The semantic markup is layered with external technologies to enhance the user experience (specifically, CSS and JavaScript), without necessarily effecting the underlying funcitonality of the document and site.</p>
>
> Commenting via email reply. Getting booted every time I try to sign in.
> Also, every time I try to update the PR via branch commit - the PR is not
> updated.(
> https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request <
> https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request>).
> This was the same workflow used for the aria-hidden PR we did for #1160
> <https://github.com/18F/web-design-standards/pull/1160> (#1160
> <https://github.com/18F/web-design-standards/pull/1160>). So, I’m
> wondering if, for some reason, the inline notes are causing an issue
> somehow. Or, if it is strictly a problem with sessioning - and the service
> problems they are experiencing: https://status.github.com Anyway. Have
> the changes made. They are queued. Now, we wait, I suppose.
> … <#m_-1873645326641936713_>
> On May 23, 2016, at 10:54 AM, Kate Garklavs ***@***.***> wrote: In
> docs/pages/getting-started.html <#1184 (comment)
> <https://github.com/18F/web-design-standards/pull/1184#discussion_r64236503>>:
> > @@ -52,6 +52,12 @@ <h3 class="usa-heading">For developers</h3> > > <p>The
> standards are designed to support both older and newer browsers through
> progressive enhancement, and officially support Internet Explorer 9 and up,
> and the latest versions of Chrome, Firefox, and Safari. Internet Explorer 8
> and below generally see very low usage today, and most agency websites
> should be able to safely begin support at Internet Explorer 9.</p> > >
> +<h4>Progressive enhancement:</h4> > + > +<p>Progressive enhancement is a
> web design and development strategy emphasizing accessibility through <a
> href="http://www.w3.org/html/wg/wiki/Guide/Syntax-Semantics"
> target="_blank" title="World Wide Web Consortium - Guide/Syntax and
> Semantics">semantic HTML markup</a>. Semantic HTML allows older browsers to
> present the content of a document to users. The semantic markup is layered
> with external technologies to enhance the user experience (specifically,
> CSS and JavaScript), without necessarily effecting the underlying
> funcitonality of the document and site.</p> Also! Noticed that
> functionality is misspelled — please update! — You are receiving this
> because you were mentioned. Reply to this email directly or view it on
> GitHub <
> https://github.com/18F/web-design-standards/pull/1184/files/a754fd75d31c84c48e82a6d67823100392c7968e#r64236503
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/web-design-standards/pull/1184/files/a754fd75d31c84c48e82a6d67823100392c7968e#r64244730>
>



-- 
Maya Benari
Innovation Specialist, 18F <http://18f.gsa.gov>
maya.ben-ari@gsa.gov